packages/contracts/src/contracts/current/protocol/Exchange/mixins/MSignatureValidator.sol

```diff
@@ -20,19 +20,15 @@ pragma solidity ^0.4.19;
 
 contract MSignatureValidator {
 
-    /// @dev Verifies that an order signature is valid.
-    /// @param signer address of signer.
-    /// @param hash Signed Keccak-256 hash.
-    /// @param v ECDSA signature parameter v.
-    /// @param r ECDSA signature parameters r.
-    /// @param s ECDSA signature parameters s.
+    /// @dev Verifies that a signature is valid.
+    /// @param digest Message digest that is signed.
+    /// @param signer Address of signer.
+    /// @param signature Proof of signing.
     /// @return Validity of order signature.
     function isValidSignature(
+        bytes32 digest,
```

Comment:
I don't really have a preference for `digest` over `hash`, but this arg is still named `hash` in `ISigner`.